tests/js/client/recovery/temp-storage-query-cleanup.js

```diff
@@ -61,9 +61,9 @@ function recoverySuite() {
     // //////////////////////////////////////////////////////////////////////////////
 
     testTempDirCleanupAfterQuery: function() {
-      const tempDir = fs.join(internal.options()["temp.intermediate-results-path"], "temp");
-      const tree = fs.listTree(tempDir);
-      assertEqual(tree.length, 1);
+      // const tempDir = fs.join(internal.options()["temp.intermediate-results-path"], "temp");
```

Comment:
If not necessary, the whole test function can be removed, which makes the entire test file useless.